packages/psss/src/containers/__tests__/weeklyReportsPanel.test.js

```diff
@@ -54,7 +56,13 @@ jest.mock('@tupaia/ui-components', () => ({
 }));
 
 function renderWeeklyReportsPanel() {
-  render(<WeeklyReportsPanel />, defaultState);
+  const history = createMemoryHistory();
+  render(
+    <Router history={history}>
```

Comment:
Is this replacing any existing history behaviour, or is it new?

Reply:
This is new. The tests used to not need a router but recent changes mean that it does and so the tests started failing. I only just realised because psss tests are not on CI/CD yet. There is another card for that.